srcpkgs/linux4.20/files/x86_64-dotconfig

```diff
@@ -0,0 +1,8952 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Linux/x86_64 4.20.0 Kernel Configuration
+#
+
+#
+# Compiler: gcc (GCC) 8.2.0
+#
+CONFIG_CC_IS_GCC=y
+CONFIG_GCC_VERSION=80200
+CONFIG_CLANG_VERSION=0
+CONFIG_IRQ_WORK=y
+CONFIG_BUILDTIME_EXTABLE_SORT=y
+CONFIG_THREAD_INFO_IN_TASK=y
+
+#
+# General setup
+#
+CONFIG_INIT_ENV_ARG_LIMIT=32
+# CONFIG_COMPILE_TEST is not set
+CONFIG_LOCALVERSION="_1"
+# CONFIG_LOCALVERSION_AUTO is not set
+CONFIG_BUILD_SALT="n"
```

Comment:
Why `CONFIG_BUILD_SALT="n"` ? I do not think that makes sense.

Reply:
( idem for 4.19: https://github.com/void-linux/void-packages/blob/master/srcpkgs/linux4.19/files/x86_64-dotconfig#L23 )

Reply:
carried over from 4.19. Indeed the default is to set it to an empty string, I changed it. Thanks for catching that.